core/src/main/resources/lib/form/textarea/textarea.js

```diff
@@ -48,7 +48,14 @@ Behaviour.specify("DIV.textarea-preview-container", 'textarea', 100, function (e
         showPreview.onclick = function() {
             // Several TEXTAREAs may exist if CodeMirror is enabled. The first one has reference to the CodeMirror object.
             var textarea = e.parentNode.getElementsByTagName("TEXTAREA")[0];
-            var text = textarea.codemirrorObject ? textarea.codemirrorObject.getValue() : textarea.value;
+            var text = "";
+            //Textarea object will be null if the text area is disabled.
+            if (textarea == null) {
+                textarea = e.parentNode.getElementsByTagName("pre")[0];
```

Comment:
This seems like a really brittle way to get the node to me. Isn't there any other way, maybe using a class or an ID?